README.md

```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
     On Debian/Ubuntu:
     
     ```bash
-    $ sudo apt install libpq-dev
+     sudo apt install libpq-dev
     ```
+ * Cargo dependency
+ * 
+   On Debian/Ubuntu:
+    
+    ```bash
+     sudo apt install cargo
+    ```
```

Comment:
Installing Rust compiler is the first item on the list. Rustup is more flexible way to install and manage rustc/cargo versions

Reply:
Yep but Cargo is mandatory for install diesel_cli
So he must be added to the list

Reply:
cargo gets installed when you install Rustup. It is really hard to install Rust without cargo. Thus, there is no need for this extra step.